docs/book/en/src/dev/rust/networking-https.md

```diff
@@ -0,0 +1,111 @@
+# Networking for Https
+
+The WasmEdge WASI socket API supports HTTP networking in Wasm apps. In order to achieve the goal of supporting HTTPS requests with the same API as HTTP request, we now create a Wasmedge plugin using openssl library. In this chapter, we will give the example of HTTPs requests and explain the design.
+
+## An HTTPS request example
+
+The [example source code](https://github.com/2019zhou/wasmedge_http_req/blob/zhou/httpsreq/examples/get_https.rs) for the HTTPS request is available as follows. The HTTP and https API are the same. The Err message are presented differently because the HTTP uses the rust code while the HTTPS request uses a wasmedge host function. 
+
+```rust
+use wasmedge_http_req::request;
+
+fn main() {
+    // get request
+    let mut writer = Vec::new(); //container for body of a response
+    let res = request::get("https://httpbin.org/get", &mut writer).unwrap();
+
+    println!("Status: {} {}", res.status_code(), res.reason());
+    println!("Headers {}", res.headers());
+    //println!("{}", String::from_utf8_lossy(&writer));
+
+    // head request
+    res = request::head("https://httpbin.org/head").unwrap();
+
+    println!("Status: {} {}", res.status_code(), res.reason());
+    println!("{:?}", res.headers());
+
+    // post request
+    writer = Vec::new(); //container for body of a response
+    const BODY: &[u8; 27] = b"field1=value1&field2=value2";
+    // let res = request::post("https://httpbin.org/post", BODY, &mut writer).unwrap();
```

Comment:
Please add a comment to explain why this line and line 36 need to be kept.

Reply:
I will delete line 30 and line 31. And add comments to line 36 as follows
``` 
line 36 //println!("{}", String::from_utf8_lossy(&writer));   // uncomment this line to display the content of variable writer
```